triage/triaging.rst

```diff
@@ -41,14 +33,13 @@ Helping Triage Issues
 =====================
 
 Once you know your way around how Python's source files are
-structured and you are comfortable working with patches, a great way to
+structured and you are comfortable with the workflow, a great way to
 contribute is to help triage issues. Do realize, though, that experience
 working on Python is needed in order to effectively help triage.
 
 Around the clock, new issues are being opened on the :ref:`issue tracker
 <tracker>` and existing issues are being updated. Every issue needs to be
-triaged to make sure various things are in proper order. Even without special
-privileges you can help with this process.
+triaged to make sure various things are in proper order.
```

Comment:
```suggestion
triaged to make sure everything runs smoothly.
```

IMO, at least in AmE the previous reads like "sorting"/"ordering" things, whereas this to me gives a better sense of the benefit it provides to the process